apps/src/gamelab/animationListModule.js

```diff
@@ -290,10 +294,13 @@ export function setInitialAnimationList(serializedAnimationList) {
   for (let i = 0; i < numberAnimations; i++) {
     const key = serializedAnimationList.orderedKeys[i];
     const name = serializedAnimationList.propsByKey[key].name;
-    for (let j = i + 1; j < numberAnimations; j++) {
+    for (let j = 0; j < numberAnimations; j++) {
+      if (j === i) {
+        continue;
+      }
```

Comment:
`continue` skips that iteration of the loop while `break` exits the loop so I don't think it's the same thing, but I don't think what I added is actually necessary so I'll update what you suggested.